src/catalog/postgres/pg_core_impl.cpp

```diff
@@ -1525,6 +1558,119 @@ Column PgCoreImpl::MakeColumn(storage::ProjectedRow *const pr, const storage::Pr
   return col;
 }
 
+template <typename Column, typename ClassOid, typename ColOid>
+void PgCoreImpl::CreateColumnStatistic(const common::ManagedPointer<transaction::TransactionContext> txn,
+                                            const ClassOid class_oid, const ColOid col_oid, const Column &col) {
+  // Step 1: Insert into the table
+  auto *const redo = txn->StageWrite(db_oid_, PgStatistic::STATISTIC_TABLE_OID, pg_statistic_all_cols_pri_);
+
+  // Write the attributes in the Redo Record
+  auto relid_entry = reinterpret_cast<ClassOid *>(
+      redo->Delta()->AccessForceNotNull(pg_statistic_all_cols_prm_[PgStatistic::STARELID_COL_OID]));
+  auto attnum_entry = reinterpret_cast<ColOid *>(
+      redo->Delta()->AccessForceNotNull(pg_statistic_all_cols_prm_[PgStatistic::STAATTNUM_COL_OID]));
+//  auto nullfrac_entry = reinterpret_cast<double *>(
```

Comment:
Please remove all dead code. This comment applies throughout the rest of the file and rest of the PR.

Reply:
Done